### ISSUE TITLE
update wording on custodian selection modal - follow up to 22905

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#26451]
## Description 

Follow up to #22905 

Wording on custodian selection modal should be updated to match [figma](https://github.com/brave/internal/issues/876#issuecomment-1299091320).

## Steps to Reproduce

   1. New profile 1.46.x, relaunch to pull griffin
   2. Enable rewards
   3. Open custodian selection modal

## Actual result:

<img width="918" alt="1 46 x" src="https://user-images.githubusercontent.com/28145373/199333516-06e46071-7fde-4126-aeee-92c6498f14a6.png">

## Expected result:

1. `Choose a wallet service` should be `Choose an account provider`

2. `Select a custodial wallet partner. If you already have an account, you will be asked to log in to connect.` should be `If you already have a custodial account, you'll be prompted to connect it. If you don't, you'll get the chance to create one.`

3. `Note: Your transactions will be visible to the selected wallet service once you authorize; for instance, they will be able to see the recipient and amount of your tips.` should be `Note: Your transactions will be visible to the custodian once you authorize. For instance, they will be able to see the recipient and amount of your tips.`

Wording changes can be confirmed with @Miyayes 

<img width="584" alt="figma" src="https://user-images.githubusercontent.com/28145373/199333546-8c61468c-e87b-49cb-86e0-ee4691ca648c.png">

## Reproduces how often: 

easily

## Brave version (brave://version info)

```
Brave | 1.46.87 Chromium: 107.0.5304.91 (Official Build) beta (x86_64)
-- | --
Revision | 3d5948960d62418160796d5831a4d2d7d6c90fa8-refs/branch-heads/5304@{#1097}
OS | macOS Version 12.6.1 (Build 21G217)
```

## Version/Channel Information:

- Can you reproduce this issue with the current release? n/a
- Can you reproduce this issue with the beta channel? 1.46.x yes
- Can you reproduce this issue with the nightly channel? 1.47.x yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by kjozwiak** [comment]
The above requires `1.46.94` or higher for `1.46.x` verification 👍

**Comment by MadhaviSeelam** [comment]
Verification `PASSED` using 
```
Brave | 1.46.96 Chromium: 107.0.5304.91 (Official Build) beta (64-bit)
-- | --
Revision | 3d5948960d62418160796d5831a4d2d7d6c90fa8-refs/branch-heads/5304@{#1097}
OS | Windows 11 Version 21H2 (Build 22000.1098)
```
Verified using the testplan from original STR https://github.com/brave/brave-browser/issues/26451#issue-1432046096

- Install 1.46.96
- launched Brave
- enabled rewards
- Open custodian selection modal

#### Confirmed correct/updated wording on the custodian selection modal specified in the Expected result screenshot . 

before fix (1.45.118)|after fix(1.46.96)
-----------|----------------
<img width="461" alt="image" src="https://user-images.githubusercontent.com/98358127/200403121-f2c613b4-5530-4e90-82e4-bcd9454b8417.png">|<img width="461" alt="image" src="https://user-images.githubusercontent.com/98358127/200402134-1cb68c37-6d4b-478a-883e-04b7d20a9d1a.png">
